backend.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
```

Comment:
In terraform 0.9, `terraform init` expect to be run at the root of the project where everything is located (plans,variables,...), and it also need a backend configuration file if we use another backend than local.
But for an unkowned reason, running Terraform 'operations' on subdirectories (like terraform apply plans) also complains for this missing backend configuration file.
This is why I define it in two places (root && plans), another solution would have been to reorganise the project structure (move all plans at the root) but in terraform 0.10 it should be possible to init subdirectories.